sdk/storage/azfile/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Release History
 
+## 1.1.1 (Unreleased)
```

Comment:
This should be 1.1.1-beta.1

Reply:
Our default is to not bump to a beta but to the latest patch version.